pandas/io/clipboard/__init__.py

```diff
@@ -156,7 +156,7 @@ def copy_qt(text):
         cb = app.clipboard()
         cb.setText(text)
 
-    def paste_qt() -> str:
+    def paste_qt():
```

Comment:
these annotations got removed?

Reply:
No, I just copied files without seeing that they changed.